src/entities/pipeline_run.py

```diff
@@ -50,22 +50,26 @@ def __init__(self, pipeline_run_dict: dict):
             for score_dict in pipeline_run_dict["run"]["results"]["scores"]:
                 self.scores.append(Score(score_dict))
 
-        self.init_predictions(pipeline_run_dict)
+        self.init_predictions(pipeline_run_dict, **kwargs)
 
         # These references will be dereferenced later by the loader
         # once the pipelines, problems, and datasets are available.
         self.datasets: List[Union[DocumentReference, Dataset]] = []
         for dataset_dict in pipeline_run_dict["datasets"]:
             self.datasets.append(DocumentReference(dataset_dict))
-        self.pipeline: Union[DocumentReference, Problem] = DocumentReference(
+        self.pipeline: Union[DocumentReference, Pipeline] = DocumentReference(
             pipeline_run_dict["pipeline"]
         )
         self.problem: Union[DocumentReference, Problem] = DocumentReference(
             pipeline_run_dict["problem"]
         )
 
-    def init_predictions(self, pipeline_run_dict) -> None:
+    def init_predictions(self, pipeline_run_dict, **kwargs) -> None:
```

Comment:
I think the more common convention is to document all used parameters in the function signature. Maybe we could change the function signature to something like `def init_predictions(self, pipeline_run_dict, *, run_predictions_path = DataDir.PREDICTIONS_DUMP.value, **kwargs) -> None:`

Reply:
Then we wouldn't need the `kwargs.get(...)` call.